db/migrate/20230713093745_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, name: 'author_id', on_delete: :cascade
+      t.string :title
+      t.text :text
+      t.integer :comments_count, default: 0
+      t.integer :likes_count, default: 0
+
```

Comment:
- [ ] As requested in the project requirements, column names should match the ERD diagram. Please make sure to change these column names to `comments_counter` and `likes_counter` as on the ERD diagram.

Reply:
Ok